commands/core/site_install.drush.inc

```diff
@@ -289,4 +300,13 @@ function drush_core_site_install($profile = NULL) {
   }
   drush_include_engine('drupal', 'site_install');
   drush_core_site_install_version($profile, $form_options);
+
+  // Post installation run the configuration import.
+  if ($config = drush_get_option('config')) {
+    // Set the destination site UUID to match the source UUID, to bypass a core fail-safe.
+    $source_storage = new FileStorage($config);
+    drush_op('drush_config_set', 'system.site', 'uuid', $source_storage->read('system.site')['uuid']);
```

Comment:
Doesn't this modify the UUID from the source config (i.e. what's on disk)? That is not a desirable behavior if the config is in version control - that config on disk should never change IMO. Since this option is only available when installing a fresh site, I think it's safe enough to change what's in the site instead of in the config directory.